Embedded System L1/AVR ATMEGA32/DIO/00-LIB/STD_TYPES.h

```diff
@@ -0,0 +1,21 @@
+/*********Author : Khalid Mosaad Zakaria */
+
+/*********Date   :   20/1/2021          */
+/*********Version: 1                     */
+/****************Desc: *******************/
+
+#ifndef	_STD_TYPES_H_
+#define _STD_TYPES_H_ 
+
+
+typedef unsigned char u8;
+typedef unsigned short int u16;
+typedef unsigned long int u32;
+typedef signed char s8;
+typedef signed short int s16;
+typedef signed long int s32;
+typedef float f32;
+typedef double f64;
```

Comment:
> -Type definition problem according to Sprints_CodeGuideLine ;
> 
> ![3](https://user-images.githubusercontent.com/44152298/127502792-a3ee8a11-ff74-4e63-b7a8-cb20d532df25.PNG)
> 
> [Sprints_CodeGuideLine.pdf](https://github.com/AhmedEssamMo/Sprints_Wave_9/files/6900596/Sprints_CodeGuideLine.pdf)

yes, You're totally right but i made it u8 because i had some issues using uint8_t for example, i solved this problem after that. 

Thank you.